boxes/identity-server-analytics/confs/repository/conf/datasources/analytics-datasources.xml

```diff
@@ -0,0 +1,111 @@
+<datasources-configuration>
+
+    <providers>
+        <provider>org.wso2.carbon.ndatasource.rdbms.RDBMSDataSourceReader</provider>
+        <!--<provider>org.wso2.carbon.datasource.reader.hadoop.HBaseDataSourceReader</provider>-->
+        <!--<provider>org.wso2.carbon.datasource.reader.cassandra.CassandraDataSourceReader</provider>-->
+    </providers>
+
+    <datasources>
+
+        <datasource>
+            <name>WSO2_ANALYTICS_EVENT_STORE_DB</name>
+            <description>The datasource used for analytics record store</description>
+            <definition type="RDBMS">
+                <configuration>
+                    <url>jdbc:mysql://172.28.128.3:3306/wso2isanalytics?autoReconnect=true</url>
+                    <username>root</username>
```

Comment:
BTW, shall we decouple Configs from been burnt into the boxes and make them available with default configs ?
We can easily mount any deployment specific configs later when we are doing that specific deployment.

Reply:
+1